packages/fx-core/resource/package.nls.json

```diff
@@ -18,10 +18,9 @@
   "core.option.learnMore": "Learn more",
   "core.option.upgrade": "Upgrade",
   "core.option.moreInfo": "More Info",
-  "core.progress.generateTemplate": "Generating Template",
-  "core.progress.generateTemplate.detail": "Downloading Template %s",
-  "core.progress.generateSample": "Generating Sample",
-  "core.progress.generateSample.detail": "Downloading Sample %s",
+  "core.progress.create": "Create",
+  "core.progress.createFromTemplate": "Downloading app template.",
```

Comment:
While we prefer displaying a user-friendly app template name in the progress bar, such as:
```
Downloading app template 'React with Fluent UI'.
```
The generator component may not be able to recognize the user-facing template name due to current code structure. If necessary, we may need to refactor the component.